<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VcsPage</class>
 <widget class="QWidget" name="VcsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>572</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Version Control Systems&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line9_3_2_2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="vcsAutoCloseCheckBox">
     <property name="text">
      <string>Close VCS dialog automatically, if no error occured</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Commit</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="vcsAutoSaveCheckBox">
        <property name="toolTip">
         <string>Select, if files should be saved before a commit</string>
        </property>
        <property name="text">
         <string>Save files upon commit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vcsAutoSaveProjectCheckBox">
        <property name="toolTip">
         <string>Select, if project should be saved before a commit</string>
        </property>
        <property name="text">
         <string>Save project upon commit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Status Monitor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QSpinBox" name="vcsStatusMonitorIntervalSpinBox">
        <property name="toolTip">
         <string>Select the interval in seconds for VCS status updates (0 to disable)</string>
        </property>
        <property name="suffix">
         <string> sec</string>
        </property>
        <property name="maximum">
         <number>3600</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="vcsMonitorLocalStatusCheckBox">
        <property name="toolTip">
         <string>Select to monitor local status only (if supported by VCS)</string>
        </property>
        <property name="text">
         <string>Monitor local status only</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="autoUpdateCheckBox">
        <property name="toolTip">
         <string>Select to enable automatic updates</string>
        </property>
        <property name="text">
         <string>Automatic updates enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Colours</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>VCS status &quot;added&quot;:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbVcsAddedButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for entries with VCS status &quot;added&quot;.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>VCS status &quot;conflict&quot;:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pbVcsConflictButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for entries with VCS status &quot;conflict&quot;.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>VCS status &quot;modified&quot;:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pbVcsModifiedButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for entries with VCS status &quot;modified&quot;.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>VCS status &quot;replaced&quot;:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pbVcsReplacedButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for entries with VCS status &quot;replaced&quot;.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>VCS status &quot;needs update&quot;:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pbVcsUpdateButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for entries with VCS status &quot;needs update&quot;.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>VCS status &quot;removed&quot;:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pbVcsRemovedButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for entries with VCS status &quot;removed&quot;.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>512</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vcsAutoCloseCheckBox</tabstop>
  <tabstop>vcsAutoSaveCheckBox</tabstop>
  <tabstop>vcsAutoSaveProjectCheckBox</tabstop>
  <tabstop>vcsStatusMonitorIntervalSpinBox</tabstop>
  <tabstop>vcsMonitorLocalStatusCheckBox</tabstop>
  <tabstop>autoUpdateCheckBox</tabstop>
  <tabstop>pbVcsAddedButton</tabstop>
  <tabstop>pbVcsModifiedButton</tabstop>
  <tabstop>pbVcsUpdateButton</tabstop>
  <tabstop>pbVcsConflictButton</tabstop>
  <tabstop>pbVcsReplacedButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
